src/rdsn/src/utils/test/je_ctl_test.cpp

```diff
@@ -0,0 +1,108 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#ifdef DSN_USE_JEMALLOC
```

Comment:
These tests will be built and ran, maybe we have to add another test type in workflows?

Reply:
> These tests will be built and ran, maybe we have to add another test type in workflows?

Good advice, I'll add workflows of build and unit tests with jemalloc. I'll try to reduce time spent on unit tests, by just adding ones jemalloc refers to.